### DISCUSSION TITLE
MeatPack support?

### DISCUSSION THREAD

**Discussion by thisisnotfez** [mriscoc/Ender3V2S1#555] [Q&A]
I couldn't find any clear communication that meatpack support is enabled in this firmware.

Is this a design choice? If so, would it behoove the community at large to know this information?

Coming over from Jyer's which is allegedly abandoned and wanted to compare features, as I am currently using it on my E3V2.

Thank you.

**Comment by mriscoc** [comment]
At 250000 baud, Meatpack is unnecessary and it is incompatible with Binary File Transfer. But it can be enabled in source code.

**Comment by thisisnotfez** [comment]
That makes total sense. Thank you for the clarification.
